CHANGELOG.md

```diff
@@ -0,0 +1,2030 @@
+* Sphinx deprecation error
+* Displayed version in docs
+* Bad json patch usage
+* Flakey hook test
+* Misc bugs
+* Up the tests
+* Mypy testing
+* Idom install
+* Doc import source url
+* Missing find_available_port arg
+* Missing server event
+* Dark mode widget view
+* Type annotations
+* Doc build
+* Flask server
+* Component name in docs
+* Accidental hard-coded style
+* Implementation
+* Link in docs
+* Component.__repr__ bug
+* Cli test on windows
+* Docs build - already installed packages not skipped
+* Idom doc server location
+* Link after reverting to docs pre
+* Warnings on fastapi render server stop
+* Docs after import source changes
+* Non-deterministic return order in install()
+* Garbage collection issue
+* Tests using key path event targets
+* Up types
+* Snake game controls
+* Doc examples
+* Spelling errors in docs
+* Flaky test
+* Mypy issues
+* Flake8 with noqa
+* Find port func
+* Api docs
+* More doc build issues
+* Log statements
+* Spelling
+* Issue and pr links
+* Export pattern matching
+* Styles
+* Javascript-components.rst
+* 417 and 413
+* #420 warn for 'key' param in render function
+* #419 and #412
+* #429 - move client js to top of src/ dir
+* Key warnings
+* Web module tests
+* Search in docs by upgrading sphinx
+* Minor docs issue
+* Accidental new line
+* Section ref
+* Race condition in test
+* Flake8 issue
+* Model mutation in client
+* Snake game
+* Manifest
+* Js release workflow name
+* Regression from 0.33.0
+* Changelog ref
+* Default export
+* Misc refs in docs
+* Title underlines
+* Doctests
+* More selenium find_element_by_* usages
+* Docs docker + docs type annotation
+* Up docker image
+* False possitive key warning
+* Force update in example loader
+* Prettier label
+* Flake8 ellipse complaint
+* For latest mypy
+* Bug in element key identity
+* Misc things about jsonpatch correction
+* Pointer event attributes
+* Re_order example
+* F-str placeholders
+* Tests for index is default key
+* Last style complaint
+* Type annotation for exports()
+
+### Removed
+
+* Async event handler
+* Private=true
+* Ability to install modules at runtime via `idom.module`
+* Version name following "+" char
+* Extra file
+* Framework classifier
+* Clientimplementation
+* More client implementation docs
+* Sys
+* Favicon from docs dockerfile
+* Loguru and use builtin logging module
+* Codecov + update license link in readme
+* Check error logs
+* Option setter/getter with current property
+* Accidentally added files
+* Unused test files
+* Runtime build dir on re-install
+* Unused import
+* All runtime usages of npm
+* Idom install from docs dockerfile
+* Name and version from root package.json
+* Code of conduct checkbox
+* Events() object
+* Accidental test file
+* Extra usages of render_json_patch in tests
+* Model event handlers in when unmounting
+* Eval from setup.py
+* Ws alert
+* Custom templating solution
+* Requireents from manifest
+* Deprecation warning
+
+## 0.9.2 - 2020-09-04
```

Comment:
Only goes to 0.9.2? Latest version is 0.35.1: https://github.com/idom-team/idom/releases